lib/parser.js

```diff
@@ -38,15 +33,18 @@ const readParcel = (
 };
 
 const parcel = ({
-// Encode parcel structure to Buffer
-  parcelId,     // id, // parcelId: 4b
-  parcelType,   // type, // parcelType: 1b
-  compression,  // compression, // 1b: no = 0, gzip = 1
-  encoding,     // encoding,
+  // Encode parcel structure to Buffer
+  parcelId, // number, id of parcel: 4b
```

Comment:
Looking at this line, I can assume that `parcelId` will always be `4b`. Please, clarify this line.

Reply:
`4b` means that size of `parcelId` is 4 bytes

Reply:
Yes, but for person who will read only function interface comments it will look like this

Reply:
Well, it's ok. But looking at it from another sight: user don't need to know details of realization. As for me, the most appropriate variant will be remove it at all except places where max value is critical and can't be overflowed. e.g. here you have 4 bytes, so you can point out that this number should be 32-bit integer